conversions/conversions_test.go

```diff
@@ -0,0 +1,28 @@
+package conversions
+
+import "testing"
+
+//BEGIN TESTS
+
+func TestRomanToInteger(t *testing.T) {
+	for _, test := range romanToIntegerTests{
+		actual := romanToInteger(test.input)
+		if actual != test.expected {
+			t.Errorf("roman to integer test %s failed", test.name,)
+		}
+	}
+}
+
+//END TESTS
+
+//BEGIN BENCHMARKS
```

Comment:
No need to have benchmark for a single implementation, please remove all unnecessary like `BEGIN TESTS` or `END TESTS` also.